neotools/applet/manager.py

```diff
@@ -28,10 +28,11 @@ def install_applet(device, content: bytes, force=False):
     if applet_type == AppletType.REGULAR:
         header = data_from_buf(APPLET_HEADER_FORMAT, content[0: APPLET_HEADER_FORMAT['size']])
 
-        if not force:
-            applet_list = read_applet_list(device)
-            if any(header['applet_id'] == applet['applet_id'] for applet in applet_list):
+        applet_list = read_applet_list(device)
+        if any(header['applet_id'] == applet['applet_id'] for applet in applet_list):
+            if not force:
                 raise NeotoolsError(f'Applet {header["name"]} is already installed')
+            remove_applet(device, header['applet_id'])
```

Comment:
The system sometimes looks up applets by ID, so having multiple with the same means it will only find the first one. For example, I think I saw where it only called initialization routines for the first of the applets with the same ID.

I chose `-f`/`--force` to be analogous to the same option on the `cp` command, but it could be `-r`/`--replace` instead. Your call.

Reply:
Yeah, I haven't thought of `cp`. The name `force` is still fine then.